recipes/dirent-path-to-parent-path/src/workflow.ts

```diff
@@ -0,0 +1,344 @@
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { resolveBindingPath } from '@nodejs/codemod-utils/ast-grep/resolve-binding-path';
+import { removeLines } from '@nodejs/codemod-utils/ast-grep/remove-lines';
+import type { Edit, Range, SgNode, SgRoot } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+
+type BindingToReplace = {
+	node: SgNode<Js>;
+	binding: string;
+	variables: { bind: SgNode<Js>; scope: SgNode<Js> }[];
+};
+
+const handledFn = ['$.readdir', '$.readdirSync', '$.opendir'];
+
+const handledModules = ['fs', 'fs/promises'];
+
+const getScopeNode = (node: SgNode<Js>, customParent?: string) => {
```

Comment:
is this should be a utility, I mean in the utility package ?

Reply:
done